docs/source/configuration.rst

```diff
@@ -675,13 +676,13 @@ ignored until a corresponding `-- noqa:enable=<rule>[,...] | all` directive.
 .. code-block:: sql
 
     -- Ignore rule L012 from this line forward
-    SELECT col_a a FROM foo --noqa: disable=L012
+    SELECT col_a a FROM foo -- noqa: disable=L012
```

Comment:
Changed all instances of `--noqa` to `-- noqa`.

Left the test cases as they were (no harm to test both!)